code/lib/manager-api-shim/README.md

```diff
@@ -0,0 +1,16 @@
+# `@storybook/client-api` -- Deprecated Story APIs (`storiesOf`)
```

Comment:
This seems like the wrong content!

PS did we talk about maybe making a `code/deprecated` folder or similar for these shim/deprecated packages?

Reply:
Yeah, that's a good idea, to move deprecated packages to a dedicated dir.

Last time I tried adding a new to the workspace for packages to exist in, it didn't go easy.. But I think It's something I'll make an attempt at for sure, another follow-up PR 👍